Fakebook.Profile/Fakebook.Profile.UnitTests/ApiTests/ProfileControllerTests.cs

```diff
@@ -217,5 +218,44 @@ public async Task CreateProfileReturnsErrorForInvalidData()
             mockedProfileRepository.Verify(x => x.CreateProfileAsync(It.IsAny<DomainProfile>()), Times.Never);
         }
         #endregion
+
+        [Fact]
+        public async Task AddFollowerAsync()
```

Comment:
this is an integration test (real context, repo, and controller), so it shouldn't be in a unit test namespace. for now, to save time, we can just make a folder for the integration tests within this project even though the project's name isn't suitable

Reply:
Moved the test to a new folder